plugins/thunderbolt/fu-plugin-thunderbolt.c

```diff
@@ -193,7 +193,7 @@ fu_plugin_thunderbolt_rescan (FuPlugin *plugin, GError **error)
 		fu_device_add_guid (info->dev, guid_id);
 
 		/* format version */
-		version = g_strdup_printf ("%" G_GUINT32_FORMAT ".%02" G_GUINT32_FORMAT,
+		version = g_strdup_printf ("%" G_GINT32_MODIFIER "x.%02" G_GINT32_MODIFIER "x",
```

Comment:
I don't think this is the right way to force it into a hex display. I think in other parts of the code we've just used %x and then done a (guint) cast for the variable, but I'm happy to be proved wrong if you have a link :)